voorhoede/tudelft-spacefinder
# Feat/implement new design of space card

# Changes

Implements the new designs of the spaces. As not all designs are implemented yet, this is a branch that merges in a special spaces redesign branch.

- Implement redesign of cards.
- Remove `<SpaceDetailCard>` card and use the updated `<SpaceCard>` everywhere in the website.
- Add placeholder for when not image is available for the space.
- Small positioning tweaks in the building header.
- The desktop view of the card detail view will be implemented in a later PR in order to keep this PR clear.

# Associated issue

- https://trello.com/c/81Dtcfon/90-update-location-card-according-to-design
- https://trello.com/c/pFGWQrmt/78-show-number-of-seats-next-to-the-seat-indicator-icon-instead-of-a-tooltip

# How to test

1. Open preview link.
2. Check if all cards look like the design.
3. Local testing: make sure everything looks good with and without opening hours visible.

# Checklist
